src/patternfly/components/Tooltip/docs/code.md

```diff
@@ -1,22 +1,23 @@
 ## Overview
 
-A tooltip is used to provide contextual information for another component on hover.  The tooltip itself is made up of two elements: arrow and content. One of the directional modifiers (`.pf-m-left`, `.pf-m-top`, etc.) is required on the tooltip component
+A tooltip is used to provide contextual information for another component on hover.  The tooltip itself is made up of two elements: arrow and content. One of the directional modifiers (`.pf-m-left`, `.pf-m-top`, etc.) is required on the tooltip component.
 
 ## Accessibility
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `aria-describedby="[id value of applicable content]"` | `.pf-c-tooltip` | Gives the tooltip an accessible description by referring to the tooltip content that describes the primary message or purpose of the dialog. Not used if there is no static text that describes the popover. |
+| `role="tooltip"` | `.pf-c-tooltip` | Adds a tooltip role to the tooltip component. **Required**|
+| `aria-describedby="[id of .pf-c-tooltip__content]"` | `*` | Gives the element that triggers the tooltip an accessible description by referring to the `id` of the `.pf-c-tooltip__content` element in the tooltip, describing the primary message or purpose of the element. Not used if there is no static text that describes the element. |
```

Comment:
Using `aria-describedby` matches the pattern described in the [wai-aria example for tooltip](https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip), but there are two use cases that are described in the Inclusive Components article for tooltips:

- One where the tooltip provides a description using `aria-describedby`, which matches this example
- One where the [tooltip provides a label](https://inclusive-components.design/tooltips-toggletips/#tooltipasprimarylabel) using `aria-labelledby`. 

I'd like for us to be able to support both use cases. The second one will be a common use case wherever an icon is included without accompanying text. 

What are your thoughts on updating this section to something like this?

| Attribute | Applied To | Outcome |
| -- | -- | -- |
| `aria-describedby="[id of .pf-c-tooltip__content]"` or `aria-labelledby="[id of .pf-c-tooltip__content]"` | `[element that triggers the tooltip]` | Makes the contents of the tooltip accessible to assistive technologies by associating the tooltip text with the element that triggers the tooltip. Use `aria-labelledby` if the tooltip provides a label for the element. Use `aria-describedby` if the element already has a label an accessible label and it's different from the tooltip text. |

Reply:
Oh, I forgot to include "**Required**" in my suggestion above. Either `aria-labelledby` or `aria-describedby` is required when using a tooltip to make the text accessible to screen readers.

Reply:
Yes looks great! Will update.